lib/cmd/common/templates.js

```diff
@@ -88,14 +106,8 @@ function listTemplates(type, headers, fields, cb) {
   });
 }
 
-// API not there yet - currently being developed.. In the meatime we get all the templates and pluck out the requested template
-/*
- function getTemplate(type, templateId, cb) {
- common.doGetApiCall(fhreq.getFeedHenryUrl(), "box/api/templates/" + type + '/' + templateId, "Error reading Template: " + templateId, cb);
- };
- */
-
 function getTemplate(type, templateId, cb) {
+  // TODO: call single item endpoint based on templateId when available
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS1135)